### ISSUE TITLE
Problems during model training

### ISSUE THREAD

**Issue by TSAI-ChengEn** [weecology/DeepForest#663]
I've noticed that during the training configuration, a 'No validation file provided. Turning off validation loop' message appears. Will this affect my training results? 

![training error](https://github.com/weecology/DeepForest/assets/148372561/9f875fc2-dc29-4cb1-85da-6ea42711806f)
Thank you for helping me answer this question

**Comment by ethanwhite** [comment]
This won't influence the training. The warning is saying that the model doesn't have a validation file, so in concept it wouldn't compare against the validation/test set, but that is intentionally separate from the training itself.

However, you are setting a validation file, so it's definitely confusing that you get this warning - but I just checked and it's actually coming from two first two lines (the two calls to `main.deepforest()`. This means that once you define the validation file in the lower lines and run `m.create_trainer()` you will have a validation file and comparison to the validation set will work as expected. If you [set your config options by adding a `deepforest_config.yml` file](https://deepforest.readthedocs.io/en/latest/ConfigurationFile.html#config) to your working directory instead of setting them in the code then these warnings will go away, but your code will work fine the way it is written.

Let me know if you have any questions. If that all sounds OK you can go ahead and close the issue.

**Comment by TSAI-ChengEn** [comment]
I understand now, thank you so much. I'll make sure to ask you if I encounter any more problems. Thanks again.

**Comment by ethanwhite** [comment]
You're welcome! I've opened an issue for us to think about making this less confusing #664.
